src/test/java/seedu/address/model/position/PositionTest.java

```diff
@@ -2,68 +2,75 @@
-        assertFalse(Position.isValidPositionName(("Administrative Assistant ^^")));
-        assertFalse(Position.isValidPositionName(("Acc-Manager")));
-        assertFalse(Position.isValidPositionName(("")));
-        assertFalse(Position.isValidPositionName((" ")));
-        assertFalse(Position.isValidPositionName(("Administrative Manager (temp)")));
-        assertFalse(Position.isValidPositionName(("'Fun!' project manager")));
-        assertFalse(Position.isValidPositionName(("data + admin clerk")));
-
-        //valid position names
-        assertTrue(Position.isValidPositionName(("Administrative Assistant")));
-        assertTrue(Position.isValidPositionName(("Receptionist")));
-        assertTrue(Position.isValidPositionName(("Office Manager")));
-        assertTrue(Position.isValidPositionName(("Auditing Clerk")));
-        assertTrue(Position.isValidPositionName(("Bookkeeper")));
-        assertTrue(Position.isValidPositionName(("Account Executive Officer Assistant")));
-        assertTrue(Position.isValidPositionName(("Branch Manager")));
-        assertTrue(Position.isValidPositionName(("Business Manager")));
-        assertTrue(Position.isValidPositionName(("Quality Control Coordinator")));
-        assertTrue(Position.isValidPositionName(("Administrative Manager")));
-        assertTrue(Position.isValidPositionName(("Chief Executive Officer")));
-        assertTrue(Position.isValidPositionName(("Business Analyst")));
-        assertTrue(Position.isValidPositionName(("Risk Manager")));
-        assertTrue(Position.isValidPositionName(("Human Resources")));
-        assertTrue(Position.isValidPositionName(("Office Assistant")));
-        assertTrue(Position.isValidPositionName(("Secretary")));
-        assertTrue(Position.isValidPositionName(("Office Clerk")));
-        assertTrue(Position.isValidPositionName(("File Clerk")));
-        assertTrue(Position.isValidPositionName(("Account Collector")));
-        assertTrue(Position.isValidPositionName(("Administrative Specialist")));
-        assertTrue(Position.isValidPositionName(("Executive Assistant")));
-        assertTrue(Position.isValidPositionName(("Program Administrator")));
-        assertTrue(Position.isValidPositionName(("Program Manager")));
-        assertTrue(Position.isValidPositionName(("Administrative Analyst")));
-        assertTrue(Position.isValidPositionName(("Data Entry")));
+    public void equals() {
+        // same values -> returns true
+        Position adminAssistantCopy = new PositionBuilder(ADMIN_ASSISTANT).build();
+        assertTrue(ADMIN_ASSISTANT.equals(adminAssistantCopy));
+
+        // same object -> returns true
+        assertTrue(ADMIN_ASSISTANT.equals(adminAssistantCopy));
+
+        // null -> returns false
+        assertFalse(adminAssistantCopy.equals(null));
+
+        // different type -> returns false
+        assertFalse(adminAssistantCopy.equals(5));
+
+        // different person -> returns false
```

Comment:
perhaps "person" should be "position" here?

Reply:
Changes made accoordinly.